test-suite/output/bug_13238.out

```diff
@@ -0,0 +1,4 @@
+Ltac bug_13238.t1 x := replace (x x) with (x x)
+Ltac bug_13238.t2 x := case : x 
+Ltac bug_13238.t3 := by move ->
+Ltac bug_13238.t4 := congr  True 
```

Comment:
I did not realize that `Print` was displaying a fully qualified name (while `Print Ltac` does not). I don't know if this has really been intended? (In any case, this is independent of this PR.)

About the space after `case`, removing it would require some general changes in `ARGUMENT EXTEND` (more precisely in `Pptactic.pr_extend_gen` iirc).

The double-spacing after `congr` can probably be addressed by removing the two `str " "` in `pr_ssrcongrarg` though.

Reply:
I have fixed the double space in `congr`. I'd rather leave the other space issue to the experts. Should I open an issue?